src/auth/jwtOptions.ts

```diff
@@ -18,7 +18,7 @@ export const jwtOptions: Params = {
 		jwksUri: `${AUTH_SERVER_ISSUER}/protocol/openid-connect/certs`,
 		rateLimit: true,
 		jwksRequestsPerMinute: 100,
-	}) as GetVerificationKey,
+	}) as unknown as GetVerificationKey,
```

Comment:
I was puzzled by this but I'm also puzzled by my earlier comment but I see that this has been overcome by a later version of mockJWKS, hurray!